Tiltfile

```diff
@@ -22,18 +23,22 @@ settings = {
     "capi_version": "v1.6.0",
     "cert_manager_version": "v1.13.2",
     "kubernetes_version": "v1.28.0",
-    "aks_kubernetes_version": "v1.26.3",
+    "aks_kubernetes_version": "v1.27.3",
     "flatcar_version": "3374.2.1",
+    "azure_location": "eastus",
+    "control_plane_machine_count": "1",
+    "az_control_plane_machine_type": "Standard_B2s",
+    "worker_machine_count": "2",
+    "az_node_machine_type": "Standard_B2s",
+    "cluster_class_name": "default",
```

Comment:
Thanks for including these! I think many users miss setting these environment variables when they try Tilt for the first time. Just to make sure: would these override user-specified environment variables?

Reply:
> I think many users miss setting these environment variables when they try Tilt for the first time.

Good callout. Shall we include these in the documentation?
Or including this would overwhelm a user?

Reply:
> Just to make sure: would these override user-specified environment variables ?

On the contrary, the values provided here will act as defaults. User can override these values by adding key-value pair in their `tilt-settings.yaml` file. 
User defined values take precedence and the settings is updated at line https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4374/files#diff-c2ee8653e1d6b85f0aadf87cd438a9250806c052877248442be4d434cbc52425R41